examples/postman/README.md

```diff
@@ -0,0 +1,13 @@
+## Postman collection
+
+This collection is designed to help in testing the nucypher encryption workflow.  Each item is linked to environment variables that are injected into future calls.
+
+The order to call the collection is as follows
+
+- Alice/derive_policy_pubkey (create a policy)
+- Enrico/encrypt_message (encrypt some data inside the policy)
+- Bob/public_keys (Get Bob's keys so alice can grant him permission)
+- Alice/grant (Grant bob permission to see the policy's data)
+- Bob/retrieve (Get alice's data)
+
+If you try to recieve the data before you are granted access you will be returned a 500 in the current version 0.0.14-alpha
```

Comment:
```suggestion
If you try to receive the data before you are granted access you will be returned a 500.
```

Reply:
version number is wrong here - it is `v0.1.0-alpha.14`

Reply:
I think it's better to not mention a specific version. An in any case, if we leave that like that, wouldn't bumpversion update it automatically, defeating this way the original intention of the sentence?

Reply:
I can remove the version, it was more to document the return, If we add this to CI the test will say we expect a 500 for the first call and no version will be necessary.  we could assert a version is returned if wanted.

Reply:
:+1:

Reply:
@mallek Can you apply this suggestion to your PR here? We'll merge it as soon as that is in.